plugin/pkg/scheduler/factory/factory.go

```diff
@@ -237,6 +240,29 @@ func (f *ConfigFactory) CreateFromKeys(predicateKeys, priorityKeys sets.String,
 	}, nil
 }
 
+func (f *ConfigFactory) getFitSchedulerPod() *api.Pod {
+	pod := f.PodQueue.Pop().(*api.Pod)
+	if f.podFitScheduler(pod) {
+		glog.V(2).Infof("About to try and scheduler pod %v", pod.Name)
+		return pod
+	}
+	return f.getFitSchedulerPod()
+}
+
+const (
```

Comment:
I think we try to define constants at the top of the file, so I'd suggest to move this to the top of the file (right after the imports).